README.md

```diff
@@ -1,4 +1,96 @@
 # Paloma
 
+## Setting up a new chain from scratch.
+
+### Initializing a new chain
+
+Download the latest release of paloma.
+
+```bash
+wget -O - https://github.com/palomachain/paloma/releases/download/v0.0.1-alphawasmd3/paloma_0.0.1-alphawasmd3_Linux_x86_64.tar.gz | \
+tar -C /var/local -xvzf - paloma
```

Comment:
the binary is actually palomad (I made the initial mistake in the first release)

Reply:
Yep. And I can change this to use "latest".